### DISCUSSION TITLE
polar.sh - funding specific issues

### DISCUSSION THREAD

**Discussion by niccokunzmann** [collective/icalendar#697] [Ideas]
Polar.sh allows funding specific issues and saying how much money would be needed to fix those. I would like to add this to the repository. This way, people can fund others for their work.

Example:
[
![grafik](https://github.com/user-attachments/assets/ce0b2a33-7aa2-4238-9866-6fdd523d3414)
](https://github.com/niccokunzmann/open-web-calendar/issues/447)

- [ ] create a new polar organization
- [ ] add the badge automatically
- [ ] set funding goals for some issues

What are your thoughts on this?

**Comment by stevepiercy** [comment]
## Thoughts, questions, and concerns

Is there a reason you don't use GitHub for Sponsorships already?

I see that you already have set up [Open Collective](https://opencollective.com/python-icalendar). I don't see any financial contributors. What is your experience with Open Collective, including success with receiving funding through it?

What would setting up Polar do that you don't already have?

----

Polar shows USD is the only currently supported currency. Conversion to other currencies could eat up payouts.

Here are a couple of links to examples of Polar integration. It looks nice.

- https://polar.sh/serenityos/issues
- https://github.com/SerenityOS/serenity/issues/12212

----

Add:

- [ ] Create a `FUNDING.yml` file. GitHub supports this, and could be used for any sponsorship service.

----

I would prefer to see a supporting organization handle sponsorships, not just one person who signs up to manage funds. One person only does not allow for continuity and succession, if something happens to that individual.

Under the `collective` GitHub organization, you should run it by https://github.com/orgs/collective/teams/staff.

Alternatively, we can propose moving this repo under the Plone Foundation. That happens often with many `collective` projects, especially add-ons, but dependencies also.

----

This is interesting to me because I am also looking for ways to fund development for Plone. Plone already uses GitHub Sponsors.

- https://docs.github.com/en/sponsors
- https://github.com/sponsors/plone

I think I will propose using Polar to the Plone Foundation. I think the best additional feature is setting and confirming payout percentages directly to contributors https://docs.polar.sh/issue-funding/reward-contributors.

**Reply by niccokunzmann** [reply]
Yes, my idea was that e.g. we can estimate a time that it takes to work on this and then say that work on this issue costs e.g. 200$. When the funding is made, work starts and the funds are distributed when it is done. I think, in some cases, I would like to tell feature requests that they can happen much easier if some money flows. This is especially nice for contributors from countries that do not have as high wages as Europe.

Other platforms are e.g. bounty source but they do not really integrate in the same way - at least not when I had a look.

**Reply by niccokunzmann** [reply]
Hm. We just have received a 202USD donation to the open collective. Now, I wonder what to do with it.

Also, it seems polar is configured for the github.com/collective - who would I talk to about this...
Then, we can proceed with issue funding.

**Reply by stevepiercy** [reply]
> Hm. We just have received a 202USD donation to the open collective. Now, I wonder what to do with it.

You can sort Discussions and Issues by most upvotes or thumbs ups to see what the community thinks. As icalendar's primary (only?) maintainer, your opinion has greater weight.

> Also, it seems polar is configured for the github.com/collective - who would I talk to about this... Then, we can proceed with issue funding.

About what exactly? It's already configured on `collective/icalendar`. There's more you might need to do in polar.sh, though. Otherwise maybe start here https://github.com/orgs/collective/teams/staff ? I'd guess half of them have moved on.

**Reply by stevepiercy** [reply]
Did you follow this guide? https://docs.polar.sh/issue-funding/getting-started

**Reply by niccokunzmann** [reply]
Issue funding is in maintenance mode and disabled by default. They seem not to be too keen on this feature and develop product selling - my impression. I might it try as it might come in handy still. But it could also be removed at any point is my feeling.

**Reply by stevepiercy** [reply]
That's only half the story. From https://github.com/orgs/polarsource/discussions/3998:

> We intend to refactor Issue Funding (and [streamline it](https://github.com/orgs/polarsource/discussions/3001)) to be built on our core billing engine in the future

See also https://github.com/orgs/polarsource/discussions/3001#discussioncomment-11293380.
